### PR TITLE
feat: add seeds paises, estados, municipios, roles

### PR DESCRIPTION
**Pull request by robertotorres-dev**
This pr is to add seeds that helps to feed the database
Tables affected
municipios
estados
paises

### CODE REVIEW

File: packages/core/src/drivers/db/models/estado.js [hunk 1]
```diff
@@ -19,7 +19,7 @@ const EstadoSchema = {
       key: 'id',
     },
   },
-  estado: {
+  nombre: {
```
Comment (hecttoressau) [inline packages/core/src/drivers/db/models/estado.js:22]: Why did you change "estado" to "nombre"? If it's not a technical requirement, we should leave like "estado" is more expressive

Comment (robertotorres-dev) [reply packages/core/src/drivers/db/models/estado.js:22]: I think that if the table is estados, these are properties of the objec Estado, for example, name, description, location, and it is repetitive if I put estado as a property of estados. It is like a Persona has propertie nombre, and not a property called persona.

**Review by hecttoressau — APPROVED** [review]
LGTM
